doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -441,6 +441,109 @@ Here are examples:
    end
    where "n + m" := (plus n m).
 
+Enabling and disabling notations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. cmd:: {| Enable | Disable } Notation @toggle_notation
+   :name: Enable Notation; Disable Notation
+
+   .. insertprodn toggle_notation toggle_notation_flag
+
+   .. prodn::
+      toggle_notation ::= {| @string | @qualid } := @one_term {? ( {+, @toggle_notation_flag } ) } {| : no scope | : @ident }
+      | {| @string | @qualid } := @one_term {? ( {+, @toggle_notation_flag } ) }
+      | {| @string | @qualid } {? ( {+, @toggle_notation_flag } ) } {| : no scope | : @ident }
+      | {| @string | @qualid } {? ( {+, @toggle_notation_flag } ) }
+      | := @one_term {? ( {+, @toggle_notation_flag } ) } {| : no scope | : @ident }
+      | := @one_term {? ( {+, @toggle_notation_flag } ) }
+      | {? ( {+, @toggle_notation_flag } ) } {| : no scope | : @ident }
+      toggle_notation_flag ::= all
+      | only parsing
+      | only printing
+      | in custom @ident
+      | in constr
+
+   Enables or disables notations previously defined with
+   :cmd:`Notation` or :cmd:`Notation (abbreviation)`.
+   This command has no effect on parser modifications made with
+   :cmd:`Reserved Notation`.
+   When multiple
+   informations are provided, the notations enabled or
+   disabled are the ones meeting the conjunction of all constraints.
```

Comment:
```suggestion
   It has no effect on notations reserved with :cmd:`Reserved Notation`.
   When multiple clauses are provided, the notations enabled or
   disabled must satisfy all of their constraints.
```

Best not to introduce a second phrase "parser modifications"
English doesn't use "informations".  "information" is an uncountable noun.  If you want to emphasize that there are multiple items, pick a different noun.